openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -242,6 +252,10 @@ def log(self):
             self._log = Logger.get_logger(repr(self))
         return self._log
 
+    @property
+    def event_system(self):
```

Comment:
I don't think this should be public, rather implement methods for allowed public usage, which is just `add_callback` if I'm looking correctly?
```python
def add_event_callback(self, topic, callback, order=None):
    return self._event_system.add_callback(topic, callback, order=order)
```